### DISCUSSION TITLE
TouchPoint in Notice Subtype 1,2,3

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#422] [Q&A] [answered]
Good Morning lab !
Just a simple question.
I made a notice subtype 1 in the preview env ( https://enotices2.preview.ted.europa.eu/home) , succefully , OK
I saw the group TouchPoint that seems useless in notice 1 ( and in 2 and 3 as well ) but I submitted the notice.
Instead in my local application CVS prompt :
Any existing touchpoint should be associated with at least one of the following role/subrole: Additional Info Provider, Doc Provider, Employment, Environmental or Fiscal Legislation information provider, Mediator, Review Information Provider, Review Organization, Tender Evaluator or Recepient, Review Body, Review Requester
And it is right , I did not associate TouchPoint with anything , well, can you tell me please what do you do in preview env , do you delete TouchPoint data before validate the notice ? Or is it  a sort of 'hidden' association ?
Thanks and Regards
Renato

**Comment by rouschr** [comment, marked as answer]
Hello, I am not sure about this, it seems that for the moment this concerns enotices2.
So you could contact the help desk of enotices2: 
OP-TED-HELPDESK@publications.europa.eu
